tests/unit/amo/test_withExperiment.js

```diff
@@ -78,7 +81,7 @@ describe(__filename, () => {
   const render = (props = {}) => {
     const root = renderWithExperiment(props);
     // Return the wrapped instance (`SomeComponentBase`)
-    return root.dive();
+    return root.dive().dive();
```

Comment:
Can we use `shallowUntilTarget`?

Reply:
I spent a lot of time experimenting with different ways of trying to get this new setup testable, and this is the only thing I could get working. Using `shallowUntilTarget` never seems to give me a testable component. If you have a specific example of something that would work I'm happy to give it a try, but I exhausted my own options on this last week.

Reply:
It's probably fine, maybe add a comment that we `dive()` twice (one for the cookie HOC, one for connect)